### PR TITLE
Manage menu

### PR DESCRIPTION
**Pull request by Lunalo**
Updating the new dialogs

### CODE REVIEW

File: instat/dlgStack.vb [hunk 1]
```diff
@@ -32,11 +30,25 @@ Public Class dlgStack
         TestOKEnabled()
     End Sub
 
+    Private Sub initialiseDialog()
```
Comment (dannyparsons) [inline instat/dlgStack.vb]: capital I needed here.

File: instat/dlgUnstack.vb [hunk 2]
```diff
@@ -35,13 +30,22 @@ Public Class dlgUnstack
...
     End Sub
-
-    Private Sub SetDefaults()
+    Private Sub InitialiseDialog()
+        ucrFactorToUnstackReceiver.Selector = ucrSelectorForUnstack
+        ucrColumnToUnstackReceiver.Selector = ucrSelectorForUnstack
+        ucrBase.clsRsyntax.SetFunction("tidyr::spread")
         ucrFactorToUnstackReceiver.SetMeAsReceiver()
         ucrSelectorForUnstack.Reset()
+        SetNewDataFrameName(ucrSelectorForUnstack.ucrAvailableDataFrames.cboAvailableDataFrames.Text & "_Unstacked")
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: lower case "u" on "unstack" for consistency.

Comment (Lunalo) [inline instat/dlgUnstack.vb]: I have now made the changes.

Thanks

Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

File: instat/dlgUnstack.vb [hunk 3]
```diff
@@ -35,13 +30,22 @@ Public Class dlgUnstack
         'Checks if Ok can be enabled.
         TestOKEnabled()
     End Sub
+    Private Sub InitialiseDialog()
+        ucrFactorTounstackReceiver.Selector = ucrSelectorForUnstack
+        ucrColumnTounstackReceiver.Selector = ucrSelectorForUnstack
+        ucrBase.clsRsyntax.SetFunction("tidyr::spread")
+        ucrFactorTounstackReceiver.SetMeAsReceiver()
+        ucrSelectorForUnstack.Reset()
+        SetNewDataFrameName(ucrSelectorForUnstack.ucrAvailableDataFrames.cboAvailableDataFrames.Text & "_Unstacked")
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: this is still upper case

Comment (Lunalo) [inline instat/dlgUnstack.vb]: Sorry, I had changed receiver part only

I have now changed for the selector too.



Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

File: instat/dlgUnstack.vb [hunk 4]
```diff
@@ -35,13 +30,22 @@ Public Class dlgUnstack
         'Checks if Ok can be enabled.
         TestOKEnabled()
     End Sub
+    Private Sub InitialiseDialog()
+        ucrFactorTounstackReceiver.Selector = ucrSelectorForunstack
+        ucrColumnTounstackReceiver.Selector = ucrSelectorForunstack
+        ucrBase.clsRsyntax.SetFunction("tidyr::spread")
+        ucrFactorTounstackReceiver.SetMeAsReceiver()
+        ucrSelectorForunstack.Reset()
+        SetNewDataFrameName(ucrSelectorForunstack.ucrAvailableDataFrames.cboAvailableDataFrames.Text & "_Unstacked")
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: Great, but I also meant here "_Unstacked" should be "_unstacked"

**Comment by Lunalo** [comment]
Fixed now

**Pull request merged**